### DISCUSSION TITLE
Error trying to change theme

### DISCUSSION THREAD

**Discussion by jamesredwards** [NvChad/NvChad#3027] [Q&A] [answered]
Getting the following error:
```
E5108: Error executing lua: ....local/share/nvim/lazy/ui/lua/nvchad/themes/mappings.lua:81: attempt to index field 'base46' (a nil value)
stack traceback:
        ....local/share/nvim/lazy/ui/lua/nvchad/themes/mappings.lua:81: in function <....local/share/nvim/lazy/ui/lua/nvchad/themes/mappings.lua:77>
```
I'm just trying to change to one of the default themes. no customisation. the error occurs, i can exit out of the menu and it changes the theme, but doesn't persist after exiting nvim.

Everything was working fine until i did a LazySync and it seems to have updated NvChad to the new plugin format.

**Comment by siduck** [comment, marked as answer]
move your theme options to M.base46, check the 2nd pinned issue
